src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -21,17 +22,29 @@ export class CalcitePickListGroup {
    */
   @Prop({ reflect: true }) textGroupTitle: string;
 
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLElement;
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
   render() {
+    const classes = classnames(CSS.container, {
+      [CSS.indent]: this.el.querySelector("[slot=parent-item]") !== null
+    });
     return (
       <Host>
         {this.textGroupTitle ? <h3 class={CSS.heading}>{this.textGroupTitle}</h3> : null}
-        <section class={CSS.container}>
+        <slot name={"parent-item"} />
```

Comment:
nitpick: can just be `name="parent-item"` or could have a SLOTS const

Reply:
@driskull I addressed this in my PR. #783